src/www/services_ntpd.php

```diff
@@ -400,11 +401,14 @@ function removeRow() {
                       <input type="button" id="showrestrictbox" class="btn btn-default btn-xs" value="<?= html_safe(gettext('Advanced')) ?>" /> - <?=gettext("Show access restriction options");?>
                       </div>
                       <div id="showrestrict" style="display:none">
-                      <?=gettext("These options control access to NTP from the WAN."); ?>
+                      <?=gettext("These options control access to NTP from all interfaces."); ?>
```

Comment:
```suggestion
                      <?=gettext("These options control access to NTP."); ?>
```

Might be better to make it shorter?